src/rulesets/oas2/__tests__/valid-example.ts

```diff
@@ -228,7 +228,7 @@ describe('valid-example', () => {
             "/pet",
             "post",
             "parameters",
-            0,
+            "0",
```

Comment:
```suggestion
            "0",
```
This should be a number.